src/main/java/teammates/common/datatransfer/CourseRoster.java

```diff
@@ -15,12 +16,24 @@
  */
 public class CourseRoster {
 
-    Map<String, StudentAttributes> studentListByEmail = new HashMap<>();
-    Map<String, InstructorAttributes> instructorListByEmail = new HashMap<>();
+    // The name for each index of the result returned by {@link CourseRoster#getInfoForIdentifier(String)}
+    public static final int INFO_NAME = 0;
+    public static final int INFO_TEAM_NAME = 1;
+    public static final int INFO_SECTION_NAME = 2;
+
+    private final Map<String, StudentAttributes> studentListByEmail = new HashMap<>();
+    private final Map<String, InstructorAttributes> instructorListByEmail = new HashMap<>();
+    private final Map<String, List<StudentAttributes>> teamToMembersTable = new HashMap<>();
 
     public CourseRoster(List<StudentAttributes> students, List<InstructorAttributes> instructors) {
         populateStudentListByEmail(students);
         populateInstructorListByEmail(instructors);
+
+        // group students by team
+        for (StudentAttributes studentAttributes : this.getStudents()) {
```

Comment:
Apply SLAP principle here: extract this block to a private method, no matter how simple it is.